R/prepare-admin-panel.R

```diff
@@ -168,14 +168,33 @@ prepare_admin_panel_components <- function(
 
   hour_levels <- c("12am", paste0(1:11, "am"), "12pm", paste0(1:11, "pm"))
 
-  log_data <- shiny::reactive({
+  full_log_data <- shiny::reactive({
     shiny::req(input$date_from, input$date_to)
 
     data_storage$read_event_data(
       as.Date(input$date_from), as.Date(input$date_to)
     )
   })
 
+  shiny::observeEvent(full_log_data(), {
+    applications <- unique(full_log_data()$app_name)
```

Comment:
Simple ordering or applications

```suggestion
    applications <- sort(unique(full_log_data()$app_name))
```

Reply:
This might be superseeded by the other comment